user_tools/pyproject.toml

```diff
@@ -8,15 +8,23 @@ authors = [
   { name="Raza Jafri", email="raza.jafri@gmail.com" },
   { name="Ahmed Hussein", email="a@ahussein.me" },
 ]
-description = "A simple wrapper process around gcloud to process the event logs from local or GCS"
+description = "A simple wrapper process around cloud service providers to run RAPIDS Accelerator for Apache Spark."
```

Comment:
"to run RAPIDS Accelerator for Apache Spark" -> "to run tools for the RAPIDS Accelerator for Apache Spark"